### ISSUE TITLE
OS-independent temp file and other fixes.  Permission to make a pull request?

### ISSUE THREAD

**Issue by notuntoward** [glm-tools/pyglmnet#302]
While getting the master branch to run on my system, I made several changes to the code on a new branch.  Can I have permission to make a pull request?

**Comment by jasmainak** [comment]
sure, feel to make a pull request. They are always welcome when they improve the code :)

**Comment by notuntoward** [comment]
Do you have to give me permissions of some kind?  When I run this command:

`git push origin dataset_and_ex_fixes:dataset_and_ex_fixes`

I get this error messsage:

```
remote: Permission to glm-tools/pyglmnet.git denied to notuntoward.
fatal: unable to access 'https://github.com/glm-tools/pyglmnet.git/': The requested URL returned error: 403
```
where `dataset_and_ex_fixes` is the name of my branch and where `https://github.com/glm-tools/` is a URL I can visit with my browser.

**Comment by jasmainak** [comment]
You cannot push to glm-tools. You should make a fork, push to your fork and make a pull request from that.

**Comment by notuntoward** [comment]
Thanks, I'd confused what was fork and what was branch.  You should see a pull request from me now.

**Comment by jasmainak** [comment]
closed by #303
